### ISSUE TITLE
ddsim not working on CLD compact files

### ISSUE THREAD

**Issue by aciarma** [HEP-FCC/FCCDetectors#49]
Hi, I tried to run `ddsim` on the CLD compact files but it does not work.
I am sourcing the nightlies and trying to use `$FCCDETECTORS/Detector/DetFCCeeCLD/compact/FCCee_o2_v02/FCCee_o2_v02.xml` 

Is this expected? 
`ddsim` works with ILC compacts from `$lcgeo_DIR`

**Comment by BrieucF** [comment]
Hi Andrea, no it is not, though the nightlies have had a number of problems recently...  It looks like the detector builders from `FCCDetectors` are not linked properly in these relies while the one from `k4geo` probably are, which would explain why the ILC compacts work. Can you open an issue in the [key4hep-spack](https://github.com/key4hep/key4hep-spack) repository? Meanwhile, if you can, use the stable stack (`source /cvmfs/sw.hsf.org/key4hep/setup.sh`), I tried and it works. If you prefer I can open the issue, just let me know. PS: can you be a bit more specific about the error you see next time you report an issue?

**Comment by BrieucF** [comment]
Hi Andrea, I opened [key4hep-spack#499](https://github.com/key4hep/key4hep-spack/issues/499). While waiting for the problem to be fixed centrally, calling this 
`LD_LIBRARY_PATH=$FCCDETECTORS/../../lib/:$FCCDETECTORS/../../lib64/:$LD_LIBRARY_PATH`
after sourcing the nightlies should solve the problem. NB: there seem to be another problem with the edm4hep output at the moment which will be investigated.

**Comment by aciarma** [comment]
Hi Brieuc, thank you very much for opening the issue. The fix you suggested seems to work, now I am trying with other geometries in the same directory.

**Comment by andresailer** [comment]
> edm4hep output at the moment which will be investigated.

Is there a ticket for that?

**Comment by BrieucF** [comment]
> > edm4hep output at the moment which will be investigated.
> 
> Is there a ticket for that?

Yes: [key4hep-spack#500](https://github.com/key4hep/key4hep-spack/issues/500)
